serial_ard.py

```diff
@@ -38,13 +38,18 @@
 #function converts assci to readable text
 def convert (enter):
 	out=""
-	for a in range (0,len(enter)):
-
-		#if enter[a] is \n or 'enter' return string out
-		if (enter[a])==13:
-			return out
-		else:	#if enter[a] is not \n then add the text value of it to out
-			out=out+chr(enter[a])
+	#See if input is integers representing ASCII values and convert if so
```

Comment:
My Arduino supports standard strings, but I left your code for compatibility with whatever systems your using. Without my addition the program would crash with a TypeError.